lib/toast.ts

```diff
@@ -0,0 +1,125 @@
+import Toastify from 'toastify-js'
+
+declare interface Toast {
+	showToast(): void;
+	hideToast(): void;
+}
+
+class ToastType {
+	static readonly ERROR = 'toast-error';
+	static readonly WARNING = 'toast-warning';
+	static readonly INFO = 'toast-info';
+	static readonly SUCCESS = 'toast-success';
+	static readonly PERMANENT = 'toast-error';
+}
+
+export interface ToastOptions {
+	/**
+	 * Defines the timeout after which the toast is closed. Set to 0 to have a persistent toast.
+	 */
+	timeout: number;
```

Comment:
```suggestion
	timeout?: number;
```

Same for the other props. Unless they are mandatory to specify, we should make their types nullable